IPs/RSKIPxxx.md

```diff
@@ -0,0 +1,262 @@
+   bytes pmtSerialized,
+   address rskOwner);`
+
+The field rskOwner specifies the owner of the ordinal.
+
+## Changing the Ownership of an Ordinal
+
+To query who owns an ordinal, the following method is added:
+
+`address getOrdinalOwner(uint256 tId, int index)`
+
+The method will return 0x00... (the zero 20-byte address) if the ordinal does not exist.
+
+To change ownership of an ordinal, the following method must be called by the owner:
+
+`void transferOrdinal(uint256 tId, int index, address newOwner)`
+
+The method does nothing if the ordinal does not exist.
+
+## Ordinal Peg-outs
+
+To send an ordinal, the following method must be called:
+
+`void releaseOrdinal(uint256 txId, uint256 index, bytes btcAddress)`
+
+This method will queue the release of an ordinal, sending it to the btcAddress specified. The caller must be the owner of the ordinal. Every 1 hour the ordinals that are queued to be released will be processed. The bridge may attempt to create a single transaction containing all ordinal UTXOs, and all ordinal outputs, plus a single additional input used for fee payments, and a single additional output used for change.
+
+Ordinals can be sent to any Bitcoin address.
+
+## Paying Parking Fees
+Each ordinal parked stores the date when the last annual parking fee was paid. Initially this date corresponds to the date of parking. The parking expires one year later. If the parking expires and the fee has not been paid, anyone will be able to claim the ordinal by paying its parking fee. If a bridge migration event occurs, all parked ordinals that have their fee unpaid will not be migrated, and they will be lost forever.
+
+To know how much must be paid to park an ordinal, the Bridge provides the following method: 
+
+`uint getOrdinalsAnnualParkingFee()`
+
+This method returns the fee that ordinals must pay annually for being parked. The amount is given in rBTC weis.
+
+To pay the parking fee of an ordinal, the Bridge provides the following method:
+
+`void payParkingExtensionFee(uint256 tId, int index)`
+
+This method extends the parking time of the ordinal to a year starting from the current date.
+ 
+To know exactly how much you need to pay to extend the parking time, the Bridge provides the following method:
+ 
+`uint256 getOrdinalParkingExtensionFee(uint256 tId, int index)`
+
+This method returns how much the user needs to pay to extend the parking period for one year from the current date. The value is expressed in weis.
+
+To know when an ordinal parking period expires, the Bridge provides this method:
+
+`void getOrdinalParkingExpirationDate(uint256 tId, int index)`
+
+
+When paying the parking extension fee, the user should transfer the amount returned by getOrdinalParkingExtensionFee(). Any excess amount will be reimbursed to the sender by direct balance update. Note that if the user pays the parking fee several times in the same block, only the first payment will be processed. The remaining will be fully reimbursed.
```

Comment:
Could a user pay the parking fee for several years in advance?

Reply:
No. Currently the maximum that can be paid is one year.